apps/condo/domains/user/schema/SignInUserService.js

```diff
@@ -0,0 +1,233 @@
+/**
+ * Generated by `createservice user.SignInUserService --type mutations`
+ */
+
+const get = require('lodash/get')
+
+const conf = require('@open-condo/config')
+const { GQLError, GQLErrorCode: { BAD_USER_INPUT } } = require('@open-condo/keystone/errors')
+const { checkDvAndSender } = require('@open-condo/keystone/plugins/dvAndSender')
+const { GQLCustomSchema, find, getSchemaCtx, getById } = require('@open-condo/keystone/schema')
+const { extractReqLocale } = require('@open-condo/locales/extractReqLocale')
+const { i18n } = require('@open-condo/locales/loader')
+
+const { NOT_FOUND } = require('@condo/domains/common/constants/errors')
+const { DV_VERSION_MISMATCH, WRONG_FORMAT } = require('@condo/domains/common/constants/errors')
+const { normalizeEmail } = require('@condo/domains/common/utils/mail')
+const { normalizePhone } = require('@condo/domains/common/utils/phone')
+const access = require('@condo/domains/user/access/SignInUserService')
+const { SERVICE, STAFF, RESIDENT } = require('@condo/domains/user/constants/common')
+const { ConfirmPhoneAction, User } = require('@condo/domains/user/utils/serverSchema')
+
+
+/**
+ * List of possible errors, that this custom schema can throw
+ * They will be rendered in documentation section in GraphiQL for this custom schema
+ */
+const ERRORS = {
+    REQUIRED_USER_DATA_IS_MISSING: (missingFields = [], missingFieldsForUser = []) => ({
+        mutation: 'signInUser',
+        variable: ['data', 'userData'],
+        code: BAD_USER_INPUT,
+        type: 'REQUIRED_USER_DATA_IS_MISSING',
+        message: `Some required user data was missing: ${missingFields.join(', ')}`,
+        messageForUser: 'api.user.signInUser.REQUIRED_USER_DATA_IS_MISSING',
+        messageInterpolation: {
+            missingFieldsForUser: missingFieldsForUser.join(', '),
+        },
+    }),
+    NOT_AVAILABLE_FOR_USER_WITH_SERVICE_TYPE: {
+        mutation: 'signInUser',
+        variable: ['data', 'userType'],
+        code: BAD_USER_INPUT,
+        type: 'NOT_AVAILABLE_FOR_USER_WITH_SERVICE_TYPE',
+        message: 'Not available for service users',
+    },
+    DIFFERENT_PHONE_NUMBERS: {
+        mutation: 'signInUser',
+        variable: ['data', 'userData', 'phone'],
```

Comment:
Phone must not be part of `userData`.

You should always take it from ConfirmAction, so this field is redundant

Reply:
At the moment this field is really redundant.

But after adding ConfirmEmailAction, it will be needed.
It is not a fact that ConfirmEmailAction will be added in the near future, but we agreed that we would provide for this